app/routers/case_information.py

```diff
@@ -4,11 +4,13 @@
 from sqlmodel import Session, select
 from app.db import get_session
 import random
+from ..auth.security import get_current_active_user
```

Comment:
Is this relative path import needed, can we not importing it like we are doing on line 5?

Reply:
This is an explicit path import as the directory won't find it otherwise